src/pages/community/release_procedure.md

```diff
@@ -54,62 +57,90 @@ The instructions and tools obey the ASF [release policy](http://www.apache.org/l
 
 ## Release Procedure
 Simplified release procedure:
-1. Create a release branch for the target release in all git repos, such as `branch-0.8`
+1. Create a release branch for the target release in all git repos, such as `branch-1.3`
 2. Stabilize the release by fixing test failures and bugs only
-3. Tag update release for a new version to prepare a release candidate, e.g `v0.8.0-1` for RC1
+3. Tag update release for a new version to prepare a release candidate, e.g `v1.3.0-1` for RC1
```

Comment:
I'd add that there's some dependency in tagging, ie. you need to start with the SI, update the core, tag the core and finally the shim. Otherwise you'll need to re-tag dependent modules.